spec/proxytrack-test/resource_test_spec.rb

```diff
@@ -37,6 +38,12 @@
           action: [:create]
         )
       end
+      it { expect(chef_run).to create_systemd_unit('proxytrack-test.com.service') }
+      it { expect(unit).to notify('service[proxytrack-test.com]').to(:restart) }
```

Comment:
I'd prefer we go with this and remove the ``let`` statement above.
```suggestion
      it { expect(chef_run.systemd_unit('proxytrack-test.com.service')).to notify('service[proxytrack-test.com]').to(:restart) }
```